app/components/header/header.tsx

```diff
@@ -56,7 +56,7 @@ const Header = ({ categories, activeCart }: HeaderProps) => {
           </nav>
         )}
 
-        <PurchaseCart activeCart={activeCart} />
```

Comment:
I'm creating a PurchaseCartContext to centralize the active cart info and some calls that need to be called from different components